lib/elastic_apm/config.rb

```diff
@@ -210,6 +211,10 @@ def inspect
       super.split.first + '>'
     end
 
+    def default_labels
```

Comment:
Could we use the same strategy as elsewhere in Config of overriding `def default_tags=()` and `deprecate :default_tags=, :default_labels=`?

This merges on every lookup which is once per txn/span. Overriding the setter, it'd only be called once on boot.

Reply:
Making this work properly ended up being a bit tricky. I decided that instead of merging tags and labels that were set, I would always favor whichever was set last. But setting `default_tags` sets `default_labels` internally.